src/Parser/Core/Modules/Features/SpellInfo.js

```diff
@@ -15,7 +15,7 @@ import SPELLS from 'common/SPELLS';
 
 // This only works with actual healing events; casts are not recognized.
 export default {
-  [SPELLS.LIGHTS_EMBRACE_HEALING.id]: { // Sea Star of the Depthmother
+  [SPELLS.OCEANS_EMBRACE.id]: { // Sea Star of the Depthmother
```

Comment:
Wait was this wrong? are you sure? I am pretty sure I verified it working.

Reply:
Yes, this was wrong. Light embrace is a NLC trait. Oceans Embrace is the sea star procc.

Reply:
well fuck that's going to mess with holy paladin's weights